README.md

```diff
@@ -1,2 +1,21 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+
+a) The algo is describing a list comparison/duplication checking (if used in game development: possible collision avoidance--)
+
+b) The algo is moving over the list for each list item (If the algo does "n" things and each "thing is O(n) then the overall algo is O(n^2)
+
+c) 
+
+d) Because it is only checking the positions before the i (each iteration), not the whole list.
+
+e) still quadratic?? (But I did lean to O(log n))
+
+f) Python sort = O (n log n) https://www.ics.uci.edu/~pattis/ICS-33/lectures/complexitypython.txt
+
+g) linear search (worksheet algo = O (n log n) 
```

Comment:
✔️ but worth spelling out that the O(n log n) for the sorting dominates the O(n) for the linear search